### PR TITLE
initial version of UMD/TypeScript sample

### PR DESCRIPTION
**Pull request by vladima**
PR is not intended to be checked in as-is immediately - it should be updated with a proper TypeScript dependency after releasing TS 1.5.

### CODE REVIEW

File: umd/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+**Fetch dependencies:**
+```
+npm install
+```
+
+**Build and run sample in browser (using port 8080)**
+```
+npm run browser
```
Comment (mhegazy) [inline umd/README.md]: i would say it is more useful to have the commands listed out. most of the time ppl are looking at these to know how to setup things, hiding the calls to tsc would not be that helpful to them.

Comment (DanielRosenwasser) [inline umd/README.md]: I think it's nice to show you can use typescript with npm just like POJS

**Comment by vladima** [comment]
pinging @mhegazy

**Comment by mhegazy** [comment]
:+1:

**Pull request merged**